pkg/auth/requests/chained.go

```diff
@@ -16,14 +16,19 @@ type chainedAuth struct {
 	auths []Authenticator
 }
 
-func (c *chainedAuth) Authenticate(req *http.Request) (authed bool, user string, groups []string, err error) {
+func (c *chainedAuth) Authenticate(req *http.Request) (*AuthenticatorResponse, error) {
```

Comment:
Same comment as above about returning the pointer and the zero initialization below

Reply:
changed the initialization to only include the map. But not sure why pointer is a problem